src/main/java/uk/gov/hmcts/darts/transcriptions/mapper/TranscriptionResponseMapper.java

```diff
@@ -40,4 +45,31 @@ TranscriptionUrgencyResponse mapToTranscriptionUrgencyResponse(TranscriptionUrge
         transcriptionUrgencyResponse.setDescription(transcriptionUrgencyEntity.getDescription());
         return transcriptionUrgencyResponse;
     }
+
+    public static TranscriptionResponse mapToTranscriptionResponse(TranscriptionEntity transcriptionEntity) {
+
+        TranscriptionResponse transcriptionResponse = new TranscriptionResponse();
+        try {
+            CourtCaseEntity courtCase = transcriptionEntity.getCourtCase();
+            transcriptionResponse.setCaseId(courtCase.getId());
+            transcriptionResponse.setCaseNumber(courtCase.getCaseNumber());
+            transcriptionResponse.setCourthouse(courtCase.getCourthouse().getCourthouseName());
+            transcriptionResponse.setDefendants(courtCase.getDefendantStringList());
+            transcriptionResponse.setJudges(courtCase.getJudgeStringList());
+            if (!transcriptionEntity.getTranscriptionDocumentEntities().isEmpty()) {
+                transcriptionResponse.setTranscriptFileName(transcriptionEntity.getTranscriptionDocumentEntities().get(0).getFileName());
```

Comment:
This bit of the domain model is a bit complicated - should it be needed in the future to have multiple transcription documents. 
So I think you will need to do something like I've done in `downloadTranscript` - see https://github.com/hmcts/darts-api/blob/664be73e17c8d9ce196a095c7a0f58af038db858/src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java#L463 for the stream max stuff.

You're right that it might be possible not to have the 'transcriptFileName' e.g. Automated route.